GameData/SPEngine/RealPlumeConfig.cfg

```diff
@@ -0,0 +1,1066 @@
+@PART[SPEngineA4Unclad|SPEngineRD100]:AFTER[SPEngine]:NEEDS[SmokeScreen]
```

Comment:
Why does this not apply to the 'clad' A4 (the one with the boattail)?

Reply:
i haven't touched anything non ROEngine related

Reply:
Oh I see, so the clad one is already working fine?

Can we not figure out what ROE did to break all the plumes on our clones?  It'd be nicer if we could keep on cloning them, rather than reimplementing them inside SPE with this `RealPlumeConfig.cfg`.  Maybe we just need to `:AFTER` something with more Zs, if ROE isn't configuring their plumes until `:AFTER[ZZ9PluralZAlphaREALPLUME]` or something.

Reply:
i will fix most of the requested changes and then i will work on the non ROEngine stuff.